src/iam.ts

```diff
@@ -272,6 +277,27 @@ export class IAM extends IAMBase {
     throw new Error(ERROR_MESSAGES.DID_DOCUMENT_NOT_INITIALIZED);
   }
 
+  async updateAssetDidDocument(options: {
```

Comment:
I know I suggested the `updateAssetDidDocument` name but I noticed that the `IAM.publishPublicClaim()` function is shared for both assets and users. I think we should therefore reuse `updateDidDocument()` for assets so that users of `iam-client-lib` will have a more consistent API in the sense there isn't one section that has a separate function for assets and one section that has a shared function.